Formula/k/keeper-commander.rb

```diff
@@ -0,0 +1,165 @@
+class KeeperCommander < Formula
+  include Language::Python::Virtualenv
+
+  desc "Command-line and SDK interface to Keeper Password Manager"
+  homepage "https://docs.keeper.io/en/privileged-access-manager/commander-cli/overview"
+  url "https://github.com/Keeper-Security/Commander/archive/refs/tags/v17.0.6.tar.gz"
+  sha256 "2e4f933d4c2e82dc3dfa83f67e0eb2997abcb354c57d5d0f7a1c96c15cc346e8"
+  license "MIT"
+  head "https://github.com/Keeper-Security/Commander.git", branch: "master"
+
+  # bcrypt dependencies
+  depends_on "pkgconf" => :build
+  depends_on "rust" => :build
+
+  depends_on "openssl@3"
+  depends_on "python@3.13"
+
```

Comment:
uses_from_macos "libffi"

Reply:
Thanks! Shame the annotations don't show on mobile

Reply:
done